### PR TITLE
[zwavejs2mqtt] new chart

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

Next project from the zwave2mqtt maintainer, **currently in alpha!**

https://github.com/zwave-js/zwavejs2mqtt

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/zwavejs2mqtt/values.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  repository: zwavejs/zwavejs2mqtt
+  pullPolicy: IfNotPresent
+  tag: 1.0.0-alpha.2
+
+strategy:
+  type: Recreate
+
+# # See more environment varaibles in the zwavejs2mqtt documentation
+# https://zwave-js.github.io/zwavejs2mqtt/#/guide/env-vars
+env:
```
Comment (bjw-s) [inline charts/zwavejs2mqtt/values.yaml:13]:
```suggestion
env: {}
```
Might be cleaner to set `env` to an empty dict

Comment (onedr0p) [reply charts/zwavejs2mqtt/values.yaml]: damn yea. it would have errored out otherwise

File: charts/zwavejs2mqtt/values.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+image:
+  repository: zwavejs/zwavejs2mqtt
+  pullPolicy: IfNotPresent
+  tag: 1.0.0-alpha.2
+
+strategy:
+  type: Recreate
+
+# # See more environment varaibles in the zwavejs2mqtt documentation
```
Comment (bjw-s) [inline charts/zwavejs2mqtt/values.yaml:11]:
```suggestion
# # See more environment variables in the zwavejs2mqtt documentation
```
Sorry, only spotted this just now

**Review by bjw-s — APPROVED** [review]
LGTM once the checks pass 👍

**Pull request merged**